src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_ELDERLY = new Prefix("elderly/");
+    public static final Prefix PREFIX_VOLUNTEER = new Prefix("volunteer/");
```

Comment:
```suggestion
    public static final Prefix PREFIX_ELDERLY = new Prefix("ent:e/");
    public static final Prefix PREFIX_VOLUNTEER = new Prefix("ent:v/");
```
we can do this, so there will be no ambiguity between similar alphabets. i think this is better too, as it provides greater clarity on the flags we're specifying. 

perhaps we should create an issue to reorganize all the existing commands such that:
1. flags follow `flags:f/`
2. arguments follow `args:a/`
3. entities follow `ent:e/`

what do you think? others feel free to chime in

Reply:
Lets discuss this in the next meeting

Reply:
for now can you update this to be `enr/` and `vnr/` respectively?

Reply:
Fixed in commit 954655f56d0a76c3da628c0304088eb3b4dbc54a